src/main/scala/io/github/spark_redshift_community/spark/redshift/Parameters.scala

```diff
@@ -294,10 +308,24 @@ private[redshift] object Parameters {
      */
     def includeColumnList: Boolean = parameters("include_column_list").toBoolean
 
+    /**
+     * Format with which data should be unloaded by Redshift
+     * Valid values for format: csv, parquet
+     * Defaults to csv
+     */
+    def getUnloadFormat: String = parameters("unloadformat")
+
+    def getTableNameOrSubquery: String = {
+      query.map(q => s"($q)").orElse(table.map(_.toString)).get }
     /**
      * The AWS SSE-KMS key to use for encryption during UNLOAD operations
      * instead of AWS's default encryption
      */
     def sseKmsKey: Option[String] = parameters.get("sse_kms_key")
+    
+    /**
+     * The Int value to write for nulls when using CSV.
```

Comment:
Slightly confused here for the comment.
Do you intend to write, "The Int value to write for ttl when using CSV."